yarn-project/end-to-end/src/fixtures/utils.ts

```diff
@@ -172,6 +203,57 @@ export async function setupAztecRPCServer(
   };
 }
 
+/**
+ * Function to setup the test against a running sandbox.
+ * @param account - The account for use in create viem wallets.
+ * @param config - The aztec Node Configuration
+ * @param logger - The logger to be used
+ * @param numberOfAccounts - The number of accounts to create on the sandbox
+ * @returns RPC Client, viwm wallets, contract addreses etc.
+ */
+async function setupWithSandbox(
+  account: Account,
+  config: AztecNodeConfig,
+  logger: DebugLogger,
+  numberOfAccounts: number,
+) {
+  // we are setting up against the sandbox, l1 contracts are already deployed
+  const { aztecRpcServer, accounts, wallets } = await setupAztecRPCServer(numberOfAccounts, undefined, logger);
```

Comment:
I'm confused here: if we're running against the sandbox, the sandbox already runs an rpc server of its own. Why do we need to start a new one?

Reply:
I have gone through and cleaned up a few poorly named functions and functions that were 'reused' but not really reused.